models/DataObject/ClassDefinition/Service.php

```diff
@@ -358,4 +360,63 @@ public static function skipColumn($tableDefinitions, $table, $colName, $type, $d
 
         return false;
     }
+
+    /**
+     * @param array $implementsParts
+     * @param string $newInterfaces
+     * @return string|null
+     * @throws \Exception
+     */
+    public static function buildImplementsInterfaces($implementsParts = [], ?string $newInterfaces) {
```

Comment:
Again: adding a default value to a parameter makes no sense when there are subsequent parameters without default values (`$newInterfaces` is just nullable, but has no default value).

Also, it does not force the parameter type to be an array, while the method will crash if it's not an array (and even if method is called with `null` for this parameter, it will be `null` and not an empty array - the default value does not help in this case).

This also counts for `buildUseTraits()`.